masterlist.yaml

```
@@ -3732,13 +3735,36 @@ plugins:
 
 ###### Sim Settlements 2 ######
   - name: 'SS2.esm'
-    url: [ 'https://www.nexusmods.com/fallout4/mods/47976' ]
+    url:
+      - 'https://www.nexusmods.com/fallout4/mods/47976'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4183663'
+    after:
+      - 'Unofficial Fallout 4 Patch.esp'
+    req:
+      - 'HUDFramework.esm'
+      - 'WorkshopFramework.esm'
+    inc:
+      - 'SimSettlements.esm'
+      - 'SimSettlements_XPAC_RiseOfTheCommonwealth.esp'
+      - 'SimSettlements_XPAC_IndustrialRevolution.esp'
+      - 'SimSettlements_XPAC_Conqueror.esp'
```

The SS1 expansion packs all have SS1 as a master, so the one incompatibility is enough:

```suggestion
    inc: [ 'SimSettlements.esm' ]
```

---

It was a choice between the transitive closure and the "make it obvious'...

The "make it obvious" was chosen as until SS1 itself is installed, I believed the incompatibilities wouldn't be flagged?
(Leading the user to have to consume the bandwidth - these are non-trivial files - and then find out they had to throw that way as well as the other files...)